Antidote/ActiveCallViewController.m

```diff
@@ -223,12 +242,25 @@ - (void)installConstraints
         make.top.equalTo(self.topViewContainer.topMargin);
     }];
 
-    [self.tableViewOfPausedCalls makeConstraints:^(MASConstraintMaker *make) {
+    [self.pauseCallsContainer makeConstraints:^(MASConstraintMaker *make) {
         make.top.equalTo(self.topViewContainer.bottom);
         make.width.equalTo(self.view);
```

Comment:
I think that it would be better to set `left` and `right` instead of `width`. With `width` we don't have any horizontal constraints and 0 is used by default for `left`, which isn't that convenient.